website/content/sdk/guides/basics/index.md

```diff
@@ -135,6 +135,18 @@ spy_daily = openbb.stocks.load(
         monthly = True)
 ```
 
+To display charts inline, we will first need to expand the import statement block:
```

Comment:
This entire section assumes you are using a Jupyter Notebook. For example you do not need to do any of these matplotlib statements outside of the Jupyter Notebook. Please make sections for this. We have absolutely no idea how people will be using the SDK and we can't assume everyone just uses Jupyter Notebooks.

<img width="1512" alt="image" src="https://user-images.githubusercontent.com/46355364/203745482-9e636189-cf6c-40e0-9e60-bf0b5766a813.png">